dnf-behave-tests/dnf/dnf-automatic/error-report.feature

```diff
@@ -0,0 +1,25 @@
+@no_installroot
+Feature: dnf-automatic reports an error when transaction failed
+
+
+Background:
+Given I delete file "/etc/yum.repos.d/*.repo" with globs
+  And I create file "/etc/dnf/dnf.conf" with
```

Comment:
What about to add a comment why the scenario requires plugins to be switched off? I know that this is dnf test default but this is not a reason.

Reply:
This was actually copied from the other existing `dnf-automatic` test, but it is not even needed by this test. I can drop it.